cluster/addons/addon-manager/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### Version 8.7-r1  (Mon October 1 2018 Zihong Zheng <zihongz@google.com>)
```

Comment:
eh, `v8.8` was fine. wasn't sure whether it merited bumping to v9, but I guess not.

Reply:
Ah got it. Reverted to use `v8.8`.